content/en/docs/tasks/security/cert-management/custom-ca-k8s/index.md

```diff
@@ -255,6 +256,169 @@ When the workloads are deployed, above, they send CSR Requests to Istiod which f
+                      - clusterissuers.cert-manager.io/*
+                      resources:
+                      - signers
+                      verbs:
+                      - approve
+    EOF
+    $ istioctl install -f ./istio.yaml
+    {{< /text >}}
+
+1. Deploy the `proxyconfig-bar.yaml` in the bar namespace to define cert-signer for workloads.
+
+    {{< text bash >}}
+    $ cat <<EOF > ./proxyconfig-bar.yaml
+    apiVersion: networking.istio.io/v1beta1
+    kind: ProxyConfig
+    metadata:
+      name: barpc
+      namespace: bar
+    spec:
+      environmentVariables:
+        ISTIO_META_CERT_SIGNER: bar
+    EOF
+    $ kubectl apply  -f ./proxyconfig-bar.yaml
+    {{< /text >}}
+
+1. Deploy the `proxyconfig-foo.yaml` in the foo namespace to define cert-signer for workloads.
+
+    {{< text bash >}}
+    $ cat <<EOF > ./proxyconfig-bar.yaml
+    apiVersion: networking.istio.io/v1beta1
+    kind: ProxyConfig
+    metadata:
+      name: foopc
+      namespace: foo
+    spec:
+      environmentVariables:
+        ISTIO_META_CERT_SIGNER: foo
+    EOF
+    $ kubectl apply  -f ./proxyconfig-foo.yaml
+    {{< /text >}}
+
+1. Deploy the `httpbin` and `sleep` sample application in the `foo` and `bar` namespace.
+
+    {{< text bash >}}
+    $ kubectl label ns foo istio-injection=enabled
+    $ kubectl label ns bar istio-injection=enabled
+    $ kubectl apply -f samples/httpbin/httpbin.yaml -n foo
+    $ kubectl apply -f samples/sleep/sleep.yaml -n foo
+    $ kubectl apply -f samples/httpbin/httpbin.yaml -n bar
+    $ kubectl apply -f samples/sleep/sleep.yaml -n bar
+    {{< /text >}}
+
+### Verify the network connectivity between `httpbin` and `sleep` within the same namespace.
+
+When the workloads are deployed, above, they send CSR Requests with related signer info. Istiod forwards the CSR request to the custome CA for signing. The custom CA will use the correct cluster issuer to sign the cert back. Workloads under `foo` namespace will use  `foo` clusterIssuers while workloads under `bar` namespace will use the `bar` clusterIssuers. To verify that they have indeed been signed by correct clusterIssuers, We can verify workloads under the same namespace can communicate will while workloads under the different namespace will be broken.
+
+1. Check network connectivity between service `sleep` and `httpbin` under `foo` namespace.
+
+    {{< text bash >}}
+    $ export SLEEP_POD_FOO=$(kubectl get pod -n foo -l app=sleep -o jsonpath={.items..metadata.name})
+    $ kubectl exec -it $SLEEP_POD_FOO -n foo -c sleep curl http://httpbin.foo:8000/html
+    <!DOCTYPE html>
+    <html>
+      <head>
+      </head>
+      <body>
+          <h1>Herman Melville - Moby-Dick</h1>
+
+          <div>
+            <p>
+              Availing himself of the mild...
+            </p>
+          </div>
+      </body>
+     {{< /text >}}
+
+1. Check network connectivity between service `sleep`  under `foo` namespace and `httpbin` under `bar` namespace is broken.
```

Comment:
nit: rather than "broken", say that it doesn't work. 
Also, has this beed added to the external_ca e2e test?

Reply:
no, this has not been added into our external_ca e2e test yet.